.github/pull_request_template.md

```diff
@@ -1,12 +1,12 @@
 # Fixes or Implements AB#nnnn
 
 -   [ ] Enhancement
--   [ ] Bug
```

Comment:
This is incorrect actually... The heading says Fixes or Implements.  So this is not Fixes a Bugfix.

Reply:
After discussion, we've agreed to remove the checkboxes since they're redundant.